examples/BskSim/scenarios/scenario_LambertGuidance.py

```diff
@@ -0,0 +1,306 @@
+        BSK_plt.plot_surface_rel_velocity(time, r_BN_N_truth, v_BN_N_truth, sigma_PN, omega_PN_N)
+
+        figureList = {}
+        if showPlots:
+            BSK_plt.show_all_plots()
+        else:
+            fileName = os.path.basename(os.path.splitext(__file__)[0])
+            figureNames = ["orbit", "position", "velocity", "surfaceRelativeVelocity"]
+            figureList = BSK_plt.save_all_plots(fileName, figureNames)
+
+        return figureList
+
+
+def runScenario(scenario):
+
+    # Initialize simulation
+    scenario.InitializeSimulation()
+    velRef = scenario.DynModels.scObject.dynManager.getStateObject("hubVelocity")
+
+    # Configure FSW mode for first Lambert maneuver
+    scenario.modeRequest = 'lambertFirstDV'
+
+    # Configure run time and execute simulation
+    simulationTime = macros.sec2nano(scenario.tm1)
+    scenario.ConfigureStopTime(simulationTime)
+    scenario.ExecuteSimulation()
+
+    # Next, the state manager objects are called to retrieve the latest inertial position and
+    # velocity vector components:
+    vm_N = unitTestSupport.EigenVector3d2np(velRef.getState())
+
+    dv_N = scenario.dv1Cmd_recorder.dvInrtlCmd[-1, :]
+
+    # After reading the Delta-V command, the state managers velocity is updated through
+    velRef.setState(unitTestSupport.np2EigenVectorXd(vm_N + dv_N))
+    # Configure FSW mode for second Lambert maneuver
+    scenario.modeRequest = 'lambertSecondDV'
+
+    # To start up the simulation again, note that the total simulation time must be provided,
+    # not just the next incremental simulation time.
+    simulationTime = macros.sec2nano(scenario.tm2)
+    scenario.ConfigureStopTime(simulationTime)
+    scenario.ExecuteSimulation()
+
+    # Next, the state manager objects are called to retrieve the latest inertial position and
+    # velocity vector components:
+    vm_N = unitTestSupport.EigenVector3d2np(velRef.getState())
+
+    dv_N = scenario.dv2Cmd_recorder.dvInrtlCmd[-1, :]
+
+    # After reading the Delta-V command, the state managers velocity is updated through
+    velRef.setState(unitTestSupport.np2EigenVectorXd(vm_N + dv_N))
+    # disable flight software after maneuver
+    scenario.modeRequest = 'standby'
+
+    # To start up the simulation again, note that the total simulation time must be provided,
+    # not just the next incremental simulation time.
+    simulationTime = macros.sec2nano(6000.)
+    scenario.ConfigureStopTime(simulationTime)
+    scenario.ExecuteSimulation()
+
+
+def run(showPlots):
+    """
+    The scenarios can be run with the followings setups parameters:
+
+    Args:
+        showPlots (bool): Determines if the script should display plots
+
+    """
+
+    # Configure a scenario in the base simulation
+    TheScenario = scenario_LambertGuidance()
+    runScenario(TheScenario)
+    figureList = TheScenario.pull_outputs(showPlots)
+
+    return figureList
+
+
+
+
+
```

Comment:
extra lines?

Reply:
Fixed on the 270-full-dv-guidance-scenario branch (this PR is branched off 270).